### PR TITLE
Support read config file for parquet.

### PR DESCRIPTION
**Pull request by wanghaEMQ**

### CODE REVIEW

**Comment by wanghaEMQ** [comment]
Need PR https://github.com/nanomq/NanoNNG/pull/769 to be merged

**Review by JaylinYu — APPROVED** [review]

**Comment by JaylinYu** [comment]
need a macro

**Comment by wanghaEMQ** [comment]
> need a macro

Yes. of cource. done

**Review by OdyWayne — APPROVED** [review]

**Review by RanMaoyi — APPROVED** [review]

**Review by lee-emqx — APPROVED** [review]

**Review by JaylinYu — APPROVED** [review]
You can merge this first, don't mind pr

File: CMakeLists.txt [hunk 1]
```diff
@@ -201,6 +202,11 @@ if(BUILD_BENCH)
   add_definitions(-DSUPP_BENCH)
 endif()
 
+if (NNG_ENABLE_PARQUET)
```
Comment (lee-emqx) [inline CMakeLists.txt:205]: If `ENABLE_PARQUET` should be detect and set `NNG_ENABLE_PARQUET`.

Comment (wanghaEMQ) [reply CMakeLists.txt]: Good caught. fixed

**Pull request merged**